packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -880,6 +885,60 @@ describe("database test", async () => {
         });
         expect(writtenPayload.eService).toEqual(toEServiceV1(eService));
         expect(writtenPayload.descriptorId).toEqual(descriptor.id);
+        expect(deleteFile).not.toHaveBeenCalled();
+      });
+
+      it("should write on event-store for the deletion of a draft descriptor (with interface and document to delete)", async () => {
+        const deleteFile = vi.spyOn(fileManager, "deleteFile");
+        const descriptorInterface: Document = {
+          ...mockDocument,
+          id: generateId(),
+          path: "test-path1",
+        };
+        const document: Document = {
+          ...mockDocument,
+          id: generateId(),
+          path: "test-path2",
+        };
+        const descriptor: Descriptor = {
+          ...mockDescriptor,
+          interface: descriptorInterface,
+          state: descriptorState.draft,
+          docs: [document],
+        };
+        const eService: EService = {
+          ...mockEService,
+          descriptors: [descriptor],
+        };
+        await addOneEService(eService, postgresDB, eservices);
+
+        await catalogService.deleteDraftDescriptor(
+          eService.id,
+          descriptor.id,
+          getMockAuthData(eService.producerId)
+        );
+
+        const writtenEvent = await readLastEventByStreamId(
+          eService.id,
+          postgresDB
+        );
+        expect(writtenEvent.stream_id).toBe(eService.id);
+        expect(writtenEvent.version).toBe("1");
+        expect(writtenEvent.type).toBe("EServiceWithDescriptorsDeleted");
+        const writtenPayload = decodeProtobufPayload({
+          messageType: EServiceWithDescriptorsDeletedV1,
+          payload: writtenEvent.data,
+        });
+        expect(writtenPayload.eService).toEqual(toEServiceV1(eService));
+        expect(writtenPayload.descriptorId).toEqual(descriptor.id);
+        expect(deleteFile).toHaveBeenCalledWith(
+          config.storageContainer,
+          descriptorInterface.path
+        );
+        expect(deleteFile).toHaveBeenCalledWith(
+          config.storageContainer,
+          document.path
+        );
```

Comment:
I added a check to assert that the `deleteFile` is called twice: once to delete the interface and once to delete the document (indeed the `docs` array only contains one item in this case).